src/App.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
```

Comment:
Avoid importing * as much as possible, be specific with imports to avoid memory and performance issues plus help with tree-shaking in future

Reply:
Fixed, changed to import React from 'react';